gojig/go.mod

```diff
@@ -2,7 +2,7 @@ module github.com/Hipo/algojig
 
 go 1.18
 
-replace github.com/algorand/go-algorand => /Users/fergal/go-algorand
+replace github.com/algorand/go-algorand => ../../go-algorand
```

Comment:
unless you also need this for local development i suppose it can stay as is - as it will only be invoked when run via ci. Otherwise, some sort of env var here could work instead